tests/queries/0_stateless/01281_combinator_orderby.sql

```diff
@@ -0,0 +1,3 @@
+SELECT groupArrayOrderBy(2)(x, y, z) FROM (SELECT number AS x, number % 3 AS y, number % 5 AS z FROM system.numbers LIMIT 10);
+SELECT groupArrayOrderBy(1)(x, y) FROM (SELECT number AS x, number % 3 AS y FROM system.numbers_mt LIMIT 10);
+SELECT groupArrayOrderBy(1)(x, y) FROM (SELECT number AS x, number AS y FROM system.numbers_mt LIMIT 10);
```

Comment:
1) Lets add some other examples if applicable - not only `groupArray`.
2) Negative cases are also useful. Both incorrect column names, and unsupported functions.